packages/medusa/src/api-v2/store/customers/me/route.ts

```diff
@@ -3,7 +3,7 @@ import { MedusaRequest, MedusaResponse } from "../../../../types/routing"
 import { ModuleRegistrationName } from "@medusajs/modules-sdk"
 
 export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
-  const id = req.auth_user!.app_metadata?.customer_id
+  const id = req.auth?.actor_id
```

Comment:
todo: I think it comes back to the previous comment right? maybe `!` instead as this route is guarded by the auth middleware and id should be a string but not undefined right?